pkg/gardenadm/cmd/bootstrap/bootstrap.go

```diff
@@ -9,11 +9,12 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
-	"k8s.io/cli-runtime/pkg/genericiooptions"
+
+	"github.com/gardener/gardener/pkg/gardenadm/cmd"
 )
 
 // NewCommand creates a new cobra.Command.
-func NewCommand(ioStreams genericiooptions.IOStreams) *cobra.Command {
+func NewCommand(globalOpts *cmd.Options) *cobra.Command {
```

Comment:
I would embed `cmd.Options` in the command-specific `Options` type. This way, we don't need to pass both global and local options to `run`, and just use `opts.Log`.

Reply:
Nice idea, will do :)